test/functions/vass/resolution/param-detuple.good

```diff
@@ -1,11 +1,11 @@
 param-detuple.chpl:8: In function 'p2':
 param-detuple.chpl:9: warning: param proc
-  param-detuple.chpl:12: called as p2(param args(0) = "a": _any)
+  param-detuple.chpl:12: called as p2(param args(0) = "a")
 param-detuple.chpl:8: In function 'p2':
 param-detuple.chpl:9: warning: param proc
-  param-detuple.chpl:13: called as p2(param args(0) = "a": string, param args(1) = "b": string)
+  param-detuple.chpl:13: called as p2(param args(0) = "a", param args(1) = "b")
```

Comment:
Do you know what happens for param bytes arguments?

Reply:
It seems to show it as `"a": bytes`. I could make it do `b"a"`.

Reply:
I've done so and will add a test.